RELEASE/scripts/autoscend/paths/legacy_of_loathing.ash

```diff
@@ -96,16 +100,31 @@ boolean lol_buyReplicas()
 			}
 			else
 			{
-				buy($coinmaster[Replica Mr. Store], 1, $item[replica cotton candy cocoon]);
-				use(1, $item[replica cotton candy cocoon]); // put in terrarium
+				if(!is100FamRun()) //If this isn't a 100% familiar run, go ahead and get another familiar
+				{	
+					buy($coinmaster[Replica Mr. Store], 1, $item[replica cotton candy cocoon]);
+					use(1, $item[replica cotton candy cocoon]); // put in terrarium
+				}
+				else //This is a 100% familiar run, no need to buy another familiar
+				{ 
+					buy($coinmaster[Replica Mr. Store], 1, $item[Replica little box of fireworks]);
+				} 
 			}
 		}
 		else if(contains_text(page, "<b>2009</b>"))
 		{
 			if(contains_text(page, "replica apathargic bandersnatch"))
 			{
-				buy($coinmaster[Replica Mr. Store], 1, $item[replica Apathargic Bandersnatch]);
-				use(1, $item[replica Apathargic Bandersnatch]); // put in terrarium
+				if(!is100FamRun()) //If this isn't a 100% familiar run, go ahead and get another familiar
```

Comment:
With these early years, it is possible to get 2 from the same year when folks have lots of progression. Need to make sure logic works to pick 2 even in a 100 fam run. 

I suggest moving is100FamRun check to before check for bander. Then check if sunglasses are available and buy them. If they aren't available then buy bander I guess

Believe current code would break when we come through on the second time as store would contain bander so we would enter this new logic. Then we are in a 100 fam run so it would attempt to buy sunglasses again, but couldn't since there are none left